power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/SendMoneyImpl.java

```diff
@@ -1,8 +1,55 @@
 package com.adorsys.gis.powerpay.powerpaybackend.services;
-
+import com.adorsys.gis.powerpay.powerpaybackend.domain.ProcedureStatus;
+import com.adorsys.gis.powerpay.powerpaybackend.domain.Transaction;
+import com.adorsys.gis.powerpay.powerpaybackend.errorhandling.InsufficientFundsException;
+import com.adorsys.gis.powerpay.powerpaybackend.repository.MoneyTranferRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.TransactionException;
 
 @Service
 public class SendMoneyImpl implements SendMoney{
 
+    @Autowired
+    private MoneyTranferRepository moneyTranferRepository;
+
+    @Autowired
+    private Transaction transaction;
```

Comment:
Who is this bean of type `Transaction`?

Reply:
Hi @stephane-segning. Sir we don't understand your question clearly. Are you asking why we have the instance of the Transaction class as one fo the attributes?

Reply:
So the meaning of the annotation `Autowired` in Spring is to wire a Bean automatically. Why do you need that here?

Reply:
You should run your app to see the [error](https://github.com/ADORSYS-GIS/e2e-banking-app/actions/runs/8416711992/job/23043825491?pr=103).

Reply:
Alright sir. I removed the autowiring and now the test is successful.

Reply:
Good, now then also remove the object itself from there...